src/components/FileManager/FileManager.tsx

```diff
@@ -162,20 +162,8 @@ export const FileManager: React.FC<{
   const handleUpdateInline = useCallback(
     async (modelId: string, attachment: SubmissionAttachment) => {
       const figureModel = modelMap.get(modelId) as Figure
-      const imageExternalFileIndex =
```

Comment:
External file references removal is handled in LEAN-1620. It is a change across many components. Why is it handled here along with the dependencies update?

Reply:
I had to do this to get the code compile with the new transform/json-schema (which already has these things removed). Once your changes from LEAN-1620 are merged, I'll undo this change and rebase.